theano/gof/cmodule.py

```diff
@@ -2290,13 +2290,17 @@ def print_command_line_error():
 
         if status:
             print('===============================')
+            tf = tempfile.NamedTemporaryFile(delete=False)
             for i, l in enumerate(src_code.split('\n')):
                 # gcc put its messages to stderr, so we add ours now
-                print('%05i\t%s' % (i + 1, l), file=sys.stderr)
+                tf.write(str(i + 1) + '\t' + l + '\n')
```

Comment:
As the C code is now written into a file, you don't need to print line numbers at the begining of lines.